feathr_project/test/test_azure_spark_maven_e2e.py

```diff
@@ -45,7 +45,7 @@ def test_feathr_online_store_agg_features():
     if client.spark_runtime == 'databricks':
         output_path = ''.join(['dbfs:/feathrazure_cijob','_', str(now.minute), '_', str(now.second), ".avro"])
     else:
-        output_path = ''.join(['abfss://xchfeathrtest4fs@xchfeathrtest4sto.dfs.core.windows.net/demo_data/output','_', str(now.minute), '_', str(now.second), ".avro"])
+        output_path = ''.join(['abfss://feathrazuretest3fs@xchfeathrtest4sto.dfs.core.windows.net/demo_data/output','_', str(now.minute), '_', str(now.second), ".avro"])
```

Comment:
xchfeathrtest4sto - should this also be changed?

Reply:
Good catch, thanks for the comment, PR has been updated